### PR TITLE
Awesome books: a complete website with navigation: Milestone 4.

### PR DESCRIPTION
**Pull request by aliabbani**
## Awesome books: a complete website with navigation: Milestone 4.

In this milestone we:

- Modify the Awesome books application to have: navigation bar, three content sections, a books list, contact info.
- Make sure that when we click on any of the links, the main area changes to display only one of the three content sections.
- Display the current date below the navigation bar.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,25 +6,52 @@
...
+                <li>Our e-mail mail@mail.com</li>
+                <li>Our phone number: 00421987564123</li>
+                <li>Our address: Streetname 06,19921 City, Country</li>
+            </ul>
         </section>
     </main>
-    <footer></footer>
+    <footer class="copyright">
+        <span class="copyright-text">Copyright &copy; 2021 henry-kc, aliabbani</span>
+    </footer>
```
Comment (hemantso) [inline index.html:54]: - [ ] As per requirements, the footer should be settling down to the bottom of the page. But right now in our project footer is in the floating state. Kindly try to fix this so the position of the footer should be the same for all navigation links.

File: index.html [hunk 2]
```diff
@@ -6,25 +6,52 @@
...
 <body>
-    <header></header>
+    <div class="wrapper"></div>
+    <header>
+        <nav class="nav">
+            <span class="nav-title">Awesome Books</span>
+            <div class="nav-items">
+                <a class="list" href="#">List</a>|
+                <a class="add-new" href="#">Add new</a>|
+                <a class="contact" href="#">Contact</a>
```
Comment (hemantso) [inline index.html]: - [ ] Optional
When the user navigating to a specific page, we can change the colour of that nav link connected to that specific page. Let's try to improve this.

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi @aliabbani and @HENRYKC24 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Kindly try to update the live link and screenshots in your README file.

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (aliabbani) [reply index.html:54]: Alright, thank you

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**